recipes/stereo-pipeline/build.sh

```diff
@@ -0,0 +1,193 @@
+#!/bin/bash
+
+# Build dependencies first
+
+# Multiview
+cd $SRC_DIR/MultiView
+mkdir -p build && cd build
+cmake ..                                                              \
+    -DCMAKE_BUILD_TYPE=Release                                        \
+    -DMULTIVIEW_DEPS_DIR=${PREFIX}                                    \
+    -DCMAKE_OSX_DEPLOYMENT_TARGET=10.13                               \
+    -DCMAKE_MODULE_PATH=$PREFIX/share/pcl-1.13/Modules                \
```

Comment:
This seems like a hack. Use `CMAKE_ARGS` to get the `CMAKE_PREFIX_PATH`?

Reply:
Addressed.